assets/router/deployment.yaml

```diff
@@ -57,8 +61,15 @@ spec:
           - mountPath: /etc/pki/tls/private
             name: default-certificate
             readOnly: true
+          - mountPath: /etc/pki/tls/private/router-metrics
+            name: metrics-certificate
+            readOnly: true
       volumes:
       - name: default-certificate
         secret:
           defaultMode: 420
           # SecretName is set at run-time.
+      - name: metrics-certificate
+        secret:
+          defaultMode: 420
+          # SecretName is set at run-time.
```

Comment:
These are metrics related changes and we don't have to deal in this pr. 
PR https://github.com/openshift/cluster-ingress-operator/pull/108 injects similar changes in manifests.go instead of deployment.yaml mainly to enable us to turn on or off metrics feature.

Reply:
Removed.  Thanks!